RELEASE_NOTES.md

```diff
@@ -0,0 +1,12 @@
+# Frequenz Resampling Release Notes
+
+## Summary
+
+## Upgrading
+
+## New Features
+
+## Bug Fixes
+
+- Uses `DateTime::UNIX_EPOCH` instead of
+  `DateTime::from_timestamp_millis(0).unwrap()` to avoid unwrapping.
```

Comment:
A bug fix line has to be added only for fixes in released code.  Here it is not released, so this doesn't have to be mentioned.

For the first release, you just need the new features to be mentioned.